File: solutions/beecrowd/2717/2717.cpp
#include <cstdint>
#include <iostream>

int main() {
    std::int16_t n, a, b;

    while (std::cin >> n >> a >> b) {
        if (a + b <= n) {
            std::cout << "Farei hoje!" << std::endl;
        } else {
            std::cout << "Deixa para amanha!" << std::endl;
        }
    }
    return 0;
}

File: solutions/beecrowd/2717/2717.py
import sys

input = sys.stdin.readline
output = sys.stdout.write

while line := input().strip():
    n = int(line)
    a, b = map(int, input().split())

    if a + b <= n:
        output('Farei hoje!\n')
    else:
        output('Deixa para amanha!\n')
